__tests__/private.ts

```diff
@@ -547,6 +548,48 @@ describe('POST /p/newUser', () => {
       variation: 1,
     });
   });
+
+  it('should add a new user with language', async () => {
+    const { body } = await request(app.server)
+      .post('/p/newUser')
+      .set('Content-type', 'application/json')
+      .set('authorization', `Service ${process.env.ACCESS_SECRET}`)
+      .send({
+        id: usersFixture[0].id,
+        name: usersFixture[0].name,
+        image: usersFixture[0].image,
+        username: usersFixture[0].username,
+        email: usersFixture[0].email,
+        experienceLevel: 'foo',
+        language: ContentLanguage.English,
+      })
+      .expect(200);
+
+    expect(body).toEqual({ status: 'ok', userId: usersFixture[0].id });
+
+    const users = await con.getRepository(User).find({ order: { id: 'ASC' } });
+    expect(users[0].id).toEqual(usersFixture[0].id);
+    expect(users[0].language).toEqual(ContentLanguage.English);
+  });
+
+  it('should not add a new user with invalid language', async () => {
+    const { body } = await request(app.server)
+      .post('/p/newUser')
+      .set('Content-type', 'application/json')
+      .set('authorization', `Service ${process.env.ACCESS_SECRET}`)
+      .send({
+        id: usersFixture[0].id,
+        name: usersFixture[0].name,
+        image: usersFixture[0].image,
+        username: usersFixture[0].username,
+        email: usersFixture[0].email,
+        experienceLevel: 'foo',
+        language: 'klingon',
```

Comment:
😆

Reply:
until we introduce it then we will change the test 😆